apps/web/src/components/error/ErrorFallback.tsx

```
@@ -41,26 +43,56 @@ export const ErrorFallback: React.FC<ErrorFallbackProps> = ({
   const navigate = useNavigate();
   const headingRef = useRef<HTMLHeadingElement>(null);
 
+  // Safely attempt to use resilience context, fallback to default if missing
+  let resilienceContext;
+  try {
+    resilienceContext = useResilience();
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)